eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java

```diff
@@ -72,14 +76,25 @@ public void loadFromCache(CacheLoadCallback callback) {
         });
     }
 
+    protected RandomizationConfigResponse readCacheFile() throws IOException {
```

Comment:
Pulled this out so it can be mocked by tests, but it cleans things up a bit too.